src/app/zap-templates/zcl/data-model/chip/thread-border-router-management-cluster.xml

```diff
@@ -22,7 +22,7 @@ limitations under the License.
     <field name="PANChange" mask="0x1"/>
   </bitmap>
 
-  <cluster>
+  <cluster apiMaturity="provisional">
```

Comment:
Please make this XML and codegen change in a separate PR, which will land very quickly.